src/app/utils/query-url-sanitization.ts

```diff
@@ -102,7 +102,7 @@ function sanitizedQueryUrl(url: string): string {
  * @param segment
  */
 function sanitizePathSegment(previousSegment: string, segment: string): string {
-  const segmentsToIgnore = ['$value', '$count', '$ref', '$batch'];
+  const segmentsToIgnore = ['$value', '$count', '$ref', '$batch', '$metadata'];
```

Comment:
Would a check for whether the segment starts with a dollar sign help prevent growing this list?

Reply:
Yeah that's a great suggestion